tests/lib-injection/dd-lib-python-init-test-django-unsupported-package-force/validate_telemetry.py

```diff
@@ -0,0 +1,10 @@
+import json
+import sys
+
+
+in_text = sys.stdin.read()
+print(in_text)
+parsed = json.loads(in_text.split("\t")[-1])
+print(parsed)
```

Comment:
```suggestion
```

Reply:
These printed outputs are important for human auditability